src/main/java/Duke.java

```diff
@@ -1,10 +1,135 @@
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.util.ArrayList;
+
+/**
+ * Project Duke is a educational software project designed to take you
+ * through the steps of building a small software incrementally,
+ * while applying as many Java and SE techniques as possible along the way.
+ *
+ * @author AdrianOngJJ
+ * @version 0.1
+ * @since 22/1/2022
+ */
 public class Duke {
-    public static void main(String[] args) {
+    private static final String LINE_BREAK = "\n~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~";
+    private static final ArrayList<Task> masterList = new ArrayList<>();
+    /**
+     * Prints line break.
+     * @return void
```

Comment:
Should the @return tag here be deleted? It is not required here as per Oracle guidelines since the return type is void. Do refer to [Javadoc guidelines](https://www.oracle.com/sg/technical-resources/articles/java/javadoc-tool.html) for a more thorough reference for the conventions with respect to Javadoc comments.